Uni_life_tracker/app/src/main/java/com/generic/ult/LoginActivity.java

```diff
@@ -0,0 +1,119 @@
+package com.generic.ult;
+
+import Database.UserInfoDatabaseHelper;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Patterns;
+import android.widget.Button;
+import androidx.appcompat.app.AppCompatActivity;
+import com.google.android.material.textfield.TextInputLayout;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class LoginActivity  extends AppCompatActivity {
+    private static final Pattern PASSWORD_REQ =
+            Pattern.compile("^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#!$%^&+=]).{6,}$");
+
+    private TextInputLayout tiEmail;
+    private TextInputLayout tiPassword;
+    UserInfoDatabaseHelper dbhelper;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_page);
+
+
+        tiEmail = findViewById(R.id.ti_email_login);
+        tiPassword = findViewById(R.id.ti_password_login);
+
+        Button btnLogin = findViewById(R.id.btn_login);
+        btnLogin.setOnClickListener(v -> {
+
+            // If all signup credentials are correct, store the credentials
```

Comment:
signup should say login, and 'store the credentials' shouldn't be there

Reply:
fixed